lib/ansible/modules/storage/netapp/_na_cdot_aggregate.py

```diff
@@ -197,8 +197,7 @@ def apply(self):
                 changed = True
 
             elif self.state == 'present':
-                if self.name is not None and not self.name == \
-                        self.name:
+                if self.name is not None and not self.name == self.name:
```

Comment:
This conditional still looks wrong. It's always going to be false.

Reply:
Correct, I did not fix it because I don't know what the intent was.

Reply:
OK, leaving it this way is fine then, since it's functionally no different that before.

Reply:
@dagwieers this module deprecated in 2.7 so we haven't been looking at any of the issues with the deprecated modules and just pointing people at using the supported version. https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/storage/netapp/na_ontap_aggregate.py